src/ng-generate/components/shared/methods/remote-handling/requestData.ts.template

```diff
@@ -95,6 +100,14 @@ private requestData() {
                 this.dataSource.paginator = this.paginator;
             }),
         catchError((error: any) => {
+            this.dataLoadError = false;
```

Comment:
Here shouldn't be `this.dataLoadError = true`? If not, I don't see the point having this variable.

Reply:
You are right. My bad.